test/mfs/mfstest.jl

```diff
@@ -133,5 +134,24 @@ end
 
 
 
+@testset "3D symmetry" begin
+
+    ds = Systems.thomas_cyclical(b = 0.1665)
```

Comment:
We are not using predefined dynamical systems in the tests. It is not good for long term maintainance. You should explicitly construct the dynamical system.